src/seasidepropertyhandler.h

```diff
@@ -80,6 +80,8 @@ class CONTACTCACHE_EXPORT SeasidePropertyHandler : public QVersitContactHandler
     void detailProcessed(const QContact &, const QContactDetail &detail,
                          const QVersitDocument &, QSet<int> * processedFields, QList<QVersitProperty> * toBeRemoved, QList<QVersitProperty> * toBeAdded);
 
+    static void processPhoto(const QVersitProperty &property, bool *alreadyProcessed, QList<QContactDetail> * updatedDetails);
```

Comment:
I think this should be more like: "static QContactAvatar avatarFromPhotoProperty(const QVersitProperty &);"